Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -0,0 +1,387 @@
+                                            _In_ DWRITE_UNDERLINE const* underline,
+                                            _In_ IUnknown* clientDrawingEffects) throw() {
+        return S_OK;
+    };
+
+    HRESULT STDMETHODCALLTYPE DrawStrikethrough(_In_ void* clientDrawingContext,
+                                                _In_ float baselineOriginX,
+                                                _In_ float baselineOriginY,
+                                                _In_ DWRITE_STRIKETHROUGH const* strikethrough,
+                                                _In_ IUnknown* clientDrawingEffects) throw() {
+        return S_OK;
+    };
+
+    HRESULT STDMETHODCALLTYPE DrawInlineObject(_In_ void* clientDrawingContext,
+                                               _In_ float originX,
+                                               _In_ float originY,
+                                               _In_ IDWriteInlineObject* inlineObject,
+                                               _In_ BOOL isSideways,
+                                               _In_ BOOL isRightToLeft,
+                                               _In_ IUnknown* clientDrawingEffects) throw() {
+        return S_OK;
+    };
+
+    HRESULT STDMETHODCALLTYPE IsPixelSnappingDisabled(_In_opt_ void* clientDrawingContext, _Out_ BOOL* isDisabled) throw() {
+        *isDisabled = FALSE;
+        return S_OK;
+    };
+
+    HRESULT STDMETHODCALLTYPE GetCurrentTransform(_In_opt_ void* clientDrawingContext, _Out_ DWRITE_MATRIX* transform) throw() {
+        return S_OK;
+    };
+
+    HRESULT STDMETHODCALLTYPE GetPixelsPerDip(_In_opt_ void* clientDrawingContext, _Out_ float* pixelsPerDip) throw() {
+        *pixelsPerDip = 1.0;
+        return S_OK;
+    };
+};
+
+CustomDWriteTextRenderer::CustomDWriteTextRenderer() {
+}
+
+HRESULT CustomDWriteTextRenderer::RuntimeClassInitialize() {
+    return S_OK;
+}
+
+/**
+ * Helper method that will create a TextLayout and TextRenderer object with the given _CTTypesetter object, string range and frame size
+ * to render to.
+ *
+ * @parameter ts _CTTypesetter object.
+ * @parameter range string range to consider for rendering.
+ * @parameter frameSize frame constrains to render the text on.
+ * @parameter glyphDetails pointer to the _DWriteGlyphRunDetails object that contains the glyph run details that was rendeered.
+ */
+static void _GetGlyphRunDetails(_CTTypesetter* ts, CFRange range, CGRect frameSize, _DWriteGlyphRunDetails* glyphDetails) {
+    ComPtr<IDWriteTextLayout> textLayout = _createDWriteTextLayout(ts, range, frameSize);
+    ComPtr<CustomDWriteTextRenderer> textRenderer = Make<CustomDWriteTextRenderer>();
+    textLayout->Draw(glyphDetails, textRenderer.Get(), 0, 0);
+}
+
+/**
+ * Helper method to retrieve font fmaly names installed in the system.
+ *
+ * @return Unmutable array of font family name strings that are installed in the system.
+ */
+static NSArray<NSString*>* _dwriteGetFamilyNames() {
```

Comment:
Weird I fixed all of these to be compliant with our coding style. Let me make sure that change got pushed

---
In reply to: [78095476](https://github.com/Microsoft/WinObjC/pull/950#discussion_r78095476) [](ancestors = 78095476)